x-pack/plugins/lens/public/embeddable/embeddable_component.tsx

```diff
@@ -9,8 +9,8 @@ import React, { FC, useEffect } from 'react';
 import type { CoreStart, ThemeServiceStart } from 'kibana/public';
 import type { Action, UiActionsStart } from 'src/plugins/ui_actions/public';
 import type { Start as InspectorStartContract } from 'src/plugins/inspector/public';
-import { EuiLoadingChart } from '@elastic/eui';
 import {
```

Comment:
```suggestion
import { EuiLoadingChart } from '@elastic/eui';
import {
```

Restore removed line by mistake